docs/apps/authentication/authentication.md

```diff
@@ -11,6 +11,7 @@ Add `ansible_base.authentication` to your installed apps:
 INSTALLED_APPS = [
     ...
     'ansible_base.authentication',
+    'ansible_base.rbac'  # dependency
```

Comment:
I think we should either make this a hard dependency in dynamic_settings or specify here that its an optional dependency. If we are going to make it optional we should also add to this doc what it would mean if the dependency is either there or not.